tasks/syncthing.yml

```diff
@@ -0,0 +1,22 @@
+- name: Create Syncthing Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ syncthing_data_directory }}"
+
+- name: Syncthing Docker Container
+  docker_container:
+    name: syncthing
+    image: syncthing/syncthing:latest
+    pull: true
+    network_mode: host
```

Comment:
Please can you remove `network_mode: host` and replace it with the specific ports mentioned in https://github.com/syncthing/syncthing/blob/master/README-Docker.md